README.md

```diff
@@ -10,33 +12,91 @@ This is a program used to manage your qBittorrent instance such as:
 
 ## Installation
 
-#### Unraid Installation:
+* Dependencies
+  * Requires `python 3`
+
+### Unraid Installation
+
 * [Guide to setup on Unraid](https://github.com/StuffAnThings/qbit_manage/wiki/qBit-Manager-on-unRAID)
-#### Local Installation:
-* Requires `python 3`. Dependencies must be installed by running:
+
+### *Nix Installation
+
+* Download the script
+
+```bash
+wget -O qbit_manage.py 'https://raw.githubusercontent.com/StuffAnThings/qbit_manage/master/qbit_manage.py'
+```
+
+* Make it executable
+
+```bash
+chmod +x qbit_manage.py
 ```
+
+* Get & Install Requirements
+
+```bash
+wget -O requirements.txt 'https://raw.githubusercontent.com/StuffAnThings/qbit_manage/master/requirements.txt'
 pip install -r requirements.txt
 ```
-If there are issues installing dependencies try:
+
+* Get Example Config
+
+```bash
+wget -O config.yml.sample 'https://raw.githubusercontent.com/StuffAnThings/qbit_manage/master/config.yml.sample'
+```
+
+* Create Config
+
+```bash
+cp config.yml.sample config.yml
+nano -e config.yml
+```
+
+* Updating
+
+```bash
+wget -O qbit_manage.py 'https://raw.githubusercontent.com/StuffAnThings/qbit_manage/master/qbit_manage.py'
+chmod +x qbit_manage.py
+wget -O requirements.txt 'https://raw.githubusercontent.com/StuffAnThings/qbit_manage/master/requirements.txt'
+pip install -r requirements.txt
+wget -O config.yml.sample 'https://raw.githubusercontent.com/StuffAnThings/qbit_manage/master/config.yml.sample'
+dff -ui config.yml config.yml.sample
```

Comment:
ToDo: Need to think of a way to diff & merge/prompt the updated config file

maybe script in python?